elastic/apm-server
# tail sampling: Change monitoring metrics

## Motivation/summary
Introduce `sampled`, `head_unsampled` and `unprocessed` metrics for tail based sampling metrics collection.
Adapt logic for `dropped` to only contain trace events.

## Checklist





## How to test these changes
Start APM Server with tail based sampling configuration and monitoring enabled. Send requests and verify the recorded metrics. 



## Related issues

implements #6269

cc @bryce-b
